README.md

```diff
@@ -13,3 +13,31 @@ Contents of this repo:
 ## License
 
 (c) Google, 2015. Licensed under [Apache-2](LICENSE) license.
+
+## Usage
+
+First, create an unencrypted private key for use with account registration. This can be done using OpenSSL:
```

Comment:
Hm, interesting. I couldn't figure out how to get it to do that for me, it kept saying I didn't have a key. Would you be able to share the equivalent way that `goacme` does this? I'll happily set this as optional or explain how it can be done via `goacme` to keep things more self-contained.